jaqpotpy/__init__.py

```diff
@@ -1 +1,2 @@
 from .jaqpot import Jaqpot
+__version__ = '{{VERSION_PLACEHOLDER}}'
```

Comment:
@alarv i have added this line here to enable the models retrieving the jaqpotpy version during the fitting process, based on older versions. Is that ok?

Reply:
Done here: https://github.com/ntua-unit-of-control-and-informatics/jaqpotpy/pull/35/commits/7f5966a3742d75721e3821a68f095efc9c3116a5